src/week-view.c

```diff
@@ -196,44 +211,91 @@ static void week_view_draw(WeekView* wv, cairo_t* cr)
 	}
 	g_object_unref(layout);
 
-	// draw vertical lines for days
+	// vertical lines for days
 	cairo_set_dash(cr, NULL, 0, 0);
+
 	icaltimetype day = icaltime_from_timet_with_zone(wv->current_view.start, 1, wv->current_tz);
 	for (int d = 0; d < num_days; ++d, icaltime_adjust(&day, 1, 0, 0, 0)) {
 		double x = wv->x + SIDEBAR_WIDTH + d * day_width;
-		char daylabel[16];
+		char day_label[16];
 		time_t tt = icaltime_as_timet(day);
 		struct tm* t = localtime(&tt);
 
-		strftime(daylabel, 16, "%e", t);
-		cairo_move_to(cr, x + 5, wv->y + HEADER_HEIGHT - 5);
-		cairo_set_font_size(cr, 22);
-		cairo_set_source_rgb(cr, dark, dark, dark);
-		cairo_show_text(cr, daylabel);
+		// day of month
+		strftime(day_label, 16, "%e", t);
+		cairo_move_to(cr, x + 8, wv->y + HEADER_HEIGHT - 14);
+		cairo_set_font_size(cr, 14);
+
+		if (day.month == wv->now.month && day.day == wv->now.day) {
```

Comment:
Can you just use `wv->now.visible && day.day == wv->now.day` and then there is no need for the `month` variable at all? Otherwise, for completeness, you would also have to check the year, since the same day of the same month reoccurs every few years...

`visible` really means "current span includes today", perhaps a better name is needed